src/app/date-range/date-range.component.ts

```diff
@@ -63,13 +62,19 @@ export class DateRangeComponent implements OnInit {
   constructor(public settings: ApplicationSettings) {
     let today = new Date();
 
-    this.fromDate.value = new Date(today.getFullYear(), today.getMonth(), 1);
-    this.toDate.value = this.getEndOfMonth();
+    if (this.rangeWeekly) {
```

Comment:
you probably want to put this in the ngOnInit - at which point the input values will be set

Reply:
so the input value is only available there?

Reply:
yes.... I just realized it was the constructor. Thank you :) it works now